assets/src/block-templates/highlighted-cta/template.js

```diff
@@ -6,30 +6,31 @@ const template = ({
   titlePlaceholder = __('Enter text', 'planet4-blocks-backend'),
 }) => [
   [
-    'core/columns',
+    'core/group',
```

Comment:
Modify to `core/group` looks more consistent with the rest of templates. Also easier to maintain.